kfp/kfp_support_lib/shared_workflow_support/src/runtime_utils/kfp_utils.py

```diff
@@ -138,7 +138,7 @@ def default_compute_execution_params(
         cluster_gpu = w_options["replicas"] * w_options.get("gpu", 0.0)
         logger.info(f"Cluster available CPUs {cluster_cpu}, Memory {cluster_mem}, GPUs {cluster_gpu}")
         # compute number of actors
-        n_actors_cpu = int(cluster_cpu * 0.85 / a_options.get("num_cpus", 0.5))
+        n_actors_cpu = int((cluster_cpu - 1) * 0.7 / a_options.get("num_cpus", 0.5))
```

Comment:
To make it symmetric, can we try:

 n_actors_cpu = int((cluster_cpu - 1) * 0.85 / a_options.get("num_cpus", 0.5))

Reply:
noop fails with this change...

Reply:
lets put back